server/services/overviewPageService.test.ts

```diff
@@ -214,4 +231,113 @@ describe('OverviewPageService', () => {
       expect(evening[1].endTime).toEqual('19:00')
     })
   })
+
+  describe('getStatuses', () => {
```

Comment:
Nothing to be done for now - but i do wonder if this ends up being it's own service if we get much more statuses because that's going to be a lot 😅